xbmc/music/tags/TagLoaderTagLib.cpp

```diff
@@ -594,7 +594,7 @@ bool CTagLoaderTagLib::ParseXiphComment(Ogg::XiphComment *xiph, EmbeddedArt *art
     if (pictures[i].data().size())
     {
       string      mime =             pictures[i].mimeType().toCString();
-      if (mime.find("image/") != 0)
+      if (!mime.compare(0, 6, "image/"))
```

Comment:
if string starts with "image/", find would return 0 and the expression is false
compare would also return 0 if string starts with "image/" and the ! make it true